runner/server.go

```diff
@@ -216,6 +219,40 @@ func (r *TerraformRunnerServer) SetEnv(ctx context.Context, req *SetEnvRequest)
 	return &SetEnvReply{Message: "ok"}, nil
 }
 
+func (r *TerraformRunnerServer) CreateFileMappings(ctx context.Context, req *CreateFileMappingsRequest) (*CreateFileMappingsReply, error) {
+	log := ctrl.LoggerFrom(ctx).WithName(loggerName)
+	log.Info("creating file mappings")
+
+	for _, fileMapping := range req.FileMappings {
+		logMessage := fmt.Sprintf("fileMapping: content: %s, location: %s, path: %s", fileMapping.Content, fileMapping.Location, fileMapping.Path)
```

Comment:
Looks like a clear case for structured logging instead of all this string formatting